resources/faq.md

```diff
@@ -78,15 +78,11 @@ For more accurate features and issues, everything is detailed in the issues of a
 
 ## I have just updated Meilisearch, and I am getting an error: "Cannot open database, expected Meilisearch engine version..."
 
-Until our first stable release (v1.0), Meilisearch minor versions are not compatible with each other. This means **every new version is considered breaking** with the small exception of bug-fixing patches. To fix this error, simply delete your database folder (`data.ms` by default) and re-index your documents with the current-version engine. See [updating Meilisearch](/learn/advanced/updating.md) for more information.
+Meilisearch major versions introduce breaking changes and are not compatible with each other. Meilisearch version 0 releases, such as v0.30 or v0.24, are also not compatible with each other. Consult the [updating guide](/learn/advanced/updating.md) for more information on how to safely migrate your data between Meilisearch releases.
```

Comment:
Hum I'm not sure this is the right answer to the right question.

First the new error the users will get is "Error: Your database version (0.30.5) is incompatible with your current engine version (1.0.0).
To migrate data between Meilisearch versions, please follow our guide on ...:

Secondly, this is not related to major versions. Users will need to use dump (and follow the guide) even for minor version.
I'm not sure this is related to talk about breaking changes here.

Reply:
Question: do we even need this FAQ anymore? 🤔 

I started to think about how I would rewrite it... but considering how much the wording of the error has been improved, I'm not sure if the answer would really add anything!

Reply:
In the end, we'll just be explaining that you migrate your database between versions using a dump, and redirecting users to the update guide, right?

Reply:
> Question: do we even need this FAQ anymore? 🤔

I cannot answer, the stats can haha
As a developer, and also as a person doing technical I like FAQ, but only my opinion. It's cool to have a "naive" approach to the product.

> In the end, we'll just be explaining that you migrate your database between versions using a dump, and redirecting users to the update guide, right?

Yes that's only the purpose of this question.
But if you estimate the error message is clear enough from v1, maybe this question in the FAQ is not relevant anymore.